pkg/lib/message.go

```diff
@@ -24,8 +24,24 @@ type Iso8583Message interface {
 }
 
 // public functions of lib
+
+// NewISO8583MessageWithJson create data elements of message with specification with json specification file
+func NewISO8583Message(specification []byte) (Iso8583Message, error) {
```

Comment:
Jose, as it seems to me, new methods that you added here are related to spec creation, not to the message. I would put them into the `spec` (or util) package instead of `message`. In the end it may look like this:

```go
spec := spec.NewFromJson(file)
message := NewISO8583Message(spec)
```
or 
```go
spec := spec.NewWithAttributes(x, y)
message := NewISO8583Message(spec)
```

Reply:
fixed